lib/index.js

```diff
@@ -402,20 +402,26 @@ client.on('message', message => {
         // document command in help section
       case 'dotprefix':
         if (isAdmin) {
-          db.updateServerPrefix(
-            serverId, args[0]
-          ).then(data => {
-            serversConfigStore = []
-            if (data) client.guilds.cache.forEach(server => {
-              db.saveServerConfig(
-                +new Date, server.id //rework schema to eliminate the need of pushing date unecessarily here or make a find function
-              ).then(data => {
-                serversConfigStore.push(data)
+            if (args[0]?.length === 1) {
+             db.updateServerPrefix(
+               serverId, args[0]
+             ).then(data => {
+               serversConfigStore = []
+               if (data) client.guilds.cache.forEach(server => {
+                db.saveServerConfig(
+                   +new Date, server.id //rework schema to eliminate the need of pushing date unecessarily here or make a find function
+                ).then(data => {
+                   serversConfigStore.push(data)
+                })
               })
-            })
-            serversConfig = serversConfigStore;
-          })
+              serversConfig = serversConfigStore;
+              message.channel.send("Prefix changed.");
+             })
+           } else {
+             message.channel.send("Failed to change the prefix: length must have maximum length of 1 character."); 
+           }
         }
+       }
```

Comment:
This bracket must be removed; else a lot of functionality gets broken.
```suggestion
```
![image](https://user-images.githubusercontent.com/11031051/102377353-3847b080-3fcd-11eb-8fdc-bac0d5a1b7bc.png)